python_modules/dagster/dagster/_core/definitions/asset_daemon_context.py

```diff
@@ -101,6 +106,19 @@ def __init__(
             instance, asset_graph, evaluation_time=evaluation_time, logger=logger
         )
         self._data_time_resolver = CachingDataTimeResolver(self.instance_queryer)
+
+        stale_resolver = CachingStaleStatusResolver(
+            instance=instance, asset_graph=asset_graph, instance_queryer=self.instance_queryer
+        )
+        self._asset_graph_view = AssetGraphView(
+            temporal_context=TemporalContext(
+                effective_dt=self.instance_queryer.evaluation_time,
+                # TODO: we should eventually pass in an explicit last_event_id
+                last_event_id=None,
+            ),
+            stale_resolver=stale_resolver,
+        )
+        self._data_time_resolver = CachingDataTimeResolver(self.instance_queryer)
```

Comment:
What is the future of `CachingDataTimeResolver`? Is it going away with freshness policies? Or does this need to be roped into `AssetGraphView`?